scripts/build-deb.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+project_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+git_version="$(git describe --tags --always --dirty)"
+git_ref="$(echo "${git_version}" | sed 's/.*-g//' | sed 's/-dirty/DIRTY/')"
+deb_version="$(echo "${git_version}" | sed 's/^v//' | sed 's/-.*//')+git${git_ref}"
+output_dir="${project_dir}/out"
+
+echo "project root: $project_dir"
+echo "project version: $git_version"
+
+cd "$project_dir"
+mkdir -p debian
+rsync -a packaging/debian/. debian/.
+
+rm -f debian/changelog
+if [[ -z ${DEBEMAIL:-} ]]; then
+    git_user="$(git config user.name)"
+    git_email="$(git config user.email)"
+    export DEBEMAIL="${git_user} <${git_email}>"
+fi
+dch --create -v "${deb_version}" --package "azure-nvme-utils" "development build: ${git_version}"
```

Comment:
`dch` looks at the Maintainer and Uploader fields in debian/control to determine whether a given change is being made by a maintainer or not. (see [the docs](https://manpages.debian.org/bookworm/devscripts/dch.1.en.html#nmu))  When I run this script, the changelog is generated with my email address and it's recognized as a non-maintainer-upload (NMU) because my address isn't listed in `debian/control`.  So the resulting packages generate a lintian warning:
```plain
W: azure-nvme-utils source: no-nmu-in-changelog [debian/changelog:1]
W: azure-nvme-utils source: source-nmu-has-incorrect-version-number 0.1.3+gitd60d0e8DIRTY [debian/changelog:1]
```

For the dev builds, we should suppress these warnings by adding the following to `packaging/debian/source/lintian-overrides`:

```
source: no-nmu-in-changelog
source: source-nmu-has-incorrect-version-number
```

Reply:
done, thanks!